packages/web/src/scss/components/UNSTABLE_Footer/index.html

```diff
@@ -0,0 +1,1026 @@
+{{#> web/layout/plain }}
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Basic Usage</h2>
+  <div class="docs-Stack">
+
+    <footer class="UNSTABLE_Footer">
+      <div class="Container">
+        <div class="Grid Grid--cols-2 Grid--tablet--cols-4">
+          <nav aria-labelledby="footer-part-one" class="mb-600 mb-tablet-800">
+            <h3 class="typography-body-large-text-bold mb-600" id="footer-part-one">
```

Comment:
I wonder if we should make IDs in this section more unique. I know they don't clash now, but they seem too global to me.

Reply:
It would clash in the IDE and I try to make all of these IDs as unique as possible. But we can discuss it later in the #1546.